venafi/resource_venafi_certificate.go

```diff
@@ -129,6 +129,18 @@ func resourceVenafiCertificate() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"valid_days": &schema.Schema{
+				Type:        schema.TypeInt,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The maximum allowed certificate validity",
```

Comment:
Let's just say "The desired certificate validity" because the issuing CA may or may not honor it for various reasons (i.e. CA doesn't support it, misconfiguration, etc.)

Reply:
fixed on the most recent commit.